File: python/20_average.py
"""Calcular el promedio de una lista — Fácil · Números

Devuelve la media aritmética de la lista. Si está vacía, devuelve 0.0.

Fuente: https://coding-challenges.dev/problems/python-calcular-promedio
"""


def average(numbers: list[float]) -> float:
    return sum(numbers) / len(numbers) if numbers else 0.0

File: typescript/20-average.ts
// Calcular el promedio de una lista — Fácil · Números
// Devuelve la media aritmética. Si la lista está vacía, devuelve 0.
// Fuente: https://coding-challenges.dev/problems/python-calcular-promedio

export function average(numbers: number[]): number {
  if (numbers.length === 0) return 0;
  return numbers.reduce((sum, number) => sum + number, 0) / numbers.length;
}
